docs/cloud/how-to-get-started-with-temporal_cloud.md

```diff
@@ -0,0 +1,99 @@
+---
+id: how-to-get-started-with-temporal-cloud
+title: How to get started with Temporal Cloud
+sidebar_label: Get started
+description: Create an account, get certificates, create a Namespace, and invite users.
+tags:
+  - how-to
+---
+
+<!--- Onboarding guide for Temporal Cloud --->
+
+To get started with Temporal Cloud, following these onboarding steps:
+
+1. [Create an account.](#create-an-account-in-temporal-cloud)
+1. [Get CA certificates.](#get-ca-certificates)
+1. [Create a Namespace.](#create-a-namespace)
+1. [Invite users.](#invite-users)
+
+## 1. Create an account in Temporal Cloud
+
+:::caution
+
+The email address you use to create an account becomes the first global admin and can assign access to all Namespaces.
+Choose wisely.
+
+:::
+
+:::info
+
+Access to [Temporal Web UI](/web-ui) is authorized via single sign-on (SSO), currently limited to Google OAuth.
+The email addresses of all members who need access to Temporal Web UI must be registered with Google.
+If you prefer to use SAML, contact Temporal Sales.
+
+:::
+
+1. Gather the following information:
+
+   - Email address
+   - Company name (the legal entity name for your company)
+   - Address of your company headquarters
+   - Name and email address of billing contact
+   - Credit card information, including billing address if not the same as the company address
+   - Optional
+     - Preferred Temporal Cloud domain
```

Comment:
what does "preferred Temporal Cloud Domain" mean?

Reply:
An excellent question. My wording is based solely on the design in Figma.

My working assumption is that the specified domain name might not be acceptable—already exists, has trademark issues, insults your mother, whatever. However, that's only an assumption; I need additional guidance.